src/main/webapp/resources/js/pages/analysis/components/AnalysisDetails.jsx

```diff
@@ -0,0 +1,197 @@
+
+        if((updatedAnalysisName  !== "") && (updatedAnalysisName !== analysisDetailsContext.analysisName)) {
+            updateAnalysis(analysisContext.analysis.identifier, updatedAnalysisName, null).then(res => {
+                showNotification({ text: res.message});
+                analysisContextUpdateSubmissionName(updatedAnalysisName);
+            });
+        }
+    }
+
+    // Update analysis priority
+    function updateAnalysisPriority(updatedPriority) {
+        updateAnalysis(analysisContext.analysis.identifier, null, updatedPriority).then(res => {
+            showNotification({ text: res.message});
+            dispatch({ type: 'UPDATED_PRIORITY', priority: updatedPriority });
+        });
+    }
+
+    // Render priorities in priority dropdown (Admins only)
+    function renderPriorities() {
+        const priorityList = [];
+
+        for(let priority of analysisDetailsContext.priorities) {
+            priorityList.push(
+              <Select.Option key={priority} value={priority}>{priority}</Select.Option>
+            )
+        }
+        return priorityList;
+    }
+
+  return (
+      <>
+          <Tabs defaultActiveKey="4"
+                tabPosition="left"
+                style={{marginLeft:150, paddingTop:25}}
+                animated={false}
+          >
+              <TabPane tab={getI18N("analysis.tab.content.analysis.details")} key="4" style={{minWidth:300}}>
+                  <Col span={12}>
+                    <h2 style={{fontWeight: "bold"}} className="spaced-bottom">{getI18N("analysis.tab.content.analysis.details")}</h2>
+
+                    <Row>
+                      <Col xs={{ span: 12, offset: 0 }} lg={{ span: 12, offset: 0 }}>
+                        <label style={{fontWeight: "bold"}}>{getI18N("analysis.tab.content.analysis.analysis-name")}</label>
+                        <Input size="large" placeholder={analysisContext.analysis.name} id="analysis-name" />
+                      </Col>
+
+                      { analysisDetailsContext.updatePermission ?
+                          <Col xs={{ span: 12, offset: 0 }} lg={{ span: 12, offset: 0 }}>
+                            <Button size="large" type="primary" className="spaced-left__sm spaced-top__lg" onClick={() => updateSubmissionName()}>{getI18N("analysis.tab.content.analysis.update.button")}</Button>
+                          </Col>
+                        : null
+                      }
+                    </Row>
+
+                    { analysisContext.isAdmin && analysisContext.analysisState == "NEW" ?
+                        <Row className="spaced-top">
+                            <Col xs={{ span: 12, offset: 0 }} lg={{ span: 12, offset: 0 }}>
+                                <label style={{fontWeight: "bold"}}>Priority</label>
+                                <Select defaultValue={analysisContext.analysis.priority} className="form-control" onChange={updateAnalysisPriority}>
```

Comment:
Why did you add the bootstrap class `form-control`?

Reply:
Fixed in 4d5b63fdf69f55254f7b8ec40e149f8fc26fcd4a